superset/db_engine_specs.py

```diff
@@ -279,33 +280,32 @@ def convert_dttm(cls, target_type, dttm):
         return "'{}'".format(dttm.strftime('%Y-%m-%d %H:%M:%S'))
 
     @classmethod
-    def fetch_result_sets(cls, db, datasource_type):
-        """Returns a list of tables [schema1.table1, schema2.table2, ...]
+    def get_all_datasource_names(cls, db, datasource_type: str) \
```

Comment:
The reason `db` wasn't type annotated is because it caused a circular import (`models.core` already has a reference to `db_engine_specs`). This should probably be refactored, but is outside the scope of this PR.